src/lib/source-handlers/azure/list-repos.ts

```diff
@@ -0,0 +1,84 @@
+import { AzureRepoData } from './types';
+import Bottleneck from 'bottleneck';
+import base64 = require('base-64');
+import * as debugLib from 'debug';
+import { getAzureToken } from './get-azure-token';
+import { getAzureOrgName } from './get-azure-orgname';
+import * as needle from 'needle';
+
+const debug = debugLib('snyk:azure');
+const azureDefaultUrl = 'https://dev.azure.com';
+
+const limiter = new Bottleneck({
+  maxConcurrent: 1,
+  minTime: 500,
+});
+
+limiter.on('failed', async (error, jobInfo) => {
+  const id = jobInfo.options.id;
+  console.warn(`Job ${id} failed: ${error}`);
+  if (jobInfo.retryCount === 0) {
+    // Here we only retry once
+    console.log(`Retrying job ${id} in 25ms!`);
```

Comment:
please use `debug` for all messages on retries, etc. See example usage in Github or other places. `console.log` is used only in a couple of places to show an overall error or result of commands.

Reply:
Will apply